marlowe-runtime/web-server-test/Spec.hs

```diff
@@ -46,15 +67,18 @@ instance Arbitrary Web.PolicyId where
 instance Arbitrary Web.Metadata where
   arbitrary = pure $ Web.Metadata Null
 
-instance Arbitrary Web.TxStatusHeader where
+instance Arbitrary Web.TxStatus where
   arbitrary = oneof
     [ pure Web.Unsigned
     , pure Web.Submitted
-    , Web.Confirmed <$> arbitrary
+    , pure Web.Confirmed
     ]
```

Comment:
I don't see this as a blocker but here's a slightly simplified version:
```haskell
arbritrary = elements
  [ Web.Unsigned
  , Web.Submitted
  , Web.Confirmed
  ]
```
[`elements :: [a] -> Gen a`](https://hackage.haskell.org/package/QuickCheck-2.14.2/docs/Test-QuickCheck.html#v:elements)

Reply:
Thanks, well spotted